lonestar/analytics/distributed/README.md

```diff
@@ -79,11 +79,19 @@ with the -help flag.
 Specifies the partitioning that you would like to use when splitting the graph
 among multiple hosts.
 
+`-exec=Sync,Async`
+
+Specifies synchronous communication (bulk-synchronous parallel where every host
+blocks for messages from other hosts at the end of a round of execution)
+or asynchronous communication (bulk-asynchronous parallel where a host does
+not have to block on messages from other hosts at the end of the round and
+may continue execution).
```

Comment:
Add a recommendation. "We recommend using Sync for 16 or fewer devices, and Async otherwise."

Reply:
Drop this. It's already handled below.